### PR TITLE
Spending of funds

### PR DESCRIPTION
**Pull request by scopatz**
@conda-forge/core

This PR falls under "Modifying the governance document", please vote and/or comment on this PR.
This PR needs 75% of core to vote yea to pass.
To vote please leave Approve (yea) or Request Changes (nay) reviews.
If you would like changes to the current language please leave a comment or push to this branch.
This vote will end on 2018/10/23.

### CODE REVIEW

**Comment by dougalsutherland** [comment]
Should we add something that makes it easy to formally *stop* spending money? for example, if we vote to fund Appveyor until the end of the migration, but we run out of money before the migration is over, it could be bad to need to wait for a core vote before canceling it.

**Comment by scopatz** [comment]
@dougalsutherland - feel free to push some text to this branch to that effect.  I think it is a good idea, to have, but even this PR is somewhat unneeded since our governance document currently says:

> All other voting items are considered to be standard and require a 50% majority to pass

Still, I think we are more comfortable with this financial stuff being explicit

**Comment by dougalsutherland** [comment]
@conda-forge/core, I added a bit about stopping recurring expenses (3fcf03c); please make sure it looks reasonable. :)

File: src/governance.rst [hunk 1]
```diff
@@ -193,7 +193,11 @@ To call for a standard vote, here is a template PR comment:
     enough in order to prevent excessive voting.  For example, recurrent
     items (such as CI) should not need to be voted on each and every month.
     Instead, they should exist for a defined period of time (e.g. until the
-    current migration ends, or for the next year).
+    current migration ends, or for the next year). For such recurring expenses,
+    the person coordinating spending the funds can choose to cancel the
+    spending if it is deemed no longer necessary or cost-effective without
+    calling another vote, although they should make reasonable efforts to
+    notify the rest of core before doing so.
```
Comment (ocefpaf) [inline src/governance.rst]: :+1:

**Comment by isuruf** [comment]
Merge now?

**Comment by dougalsutherland** [comment]
> The voting period must be open for at least one core member meeting cycle to allow for clarification questions and discussions.

By the letter, we haven't hit that yet. But since we talked about it during the last meeting immediately before the actual PR, seems okay to me.

**Comment by scopatz** [comment]
As I wrote this I approve, which means we now have 12 members approving.

**Pull request merged**